hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -419,3 +461,10 @@ genByronCommit = do
   addr <- ByronAddressInEra <$> arbitrary
   value <- genValue
   pure $ UTxO.singleton (input, TxOut addr value TxOutDatumNone ReferenceScriptNone)
+
+-- * Helpers
+
+mfail :: MonadFail m => Maybe a -> m a
```

Comment:
This one is similar to http://localhost:49513/file/nix/store/wac09x6h2rwgfy2zi08gmg954wdzz8ma-optparse-applicative-lib-optparse-applicative-0.17.0.0-haddock-doc/share/doc/optparse-applicative/html/Options-Applicative-Internal.html#v:hoistMaybe